.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
+  Thanks for sending a pull request!
+    1. If this is your first time, please read our contributor guidelines: https://cwiki.apache.org/confluence/display/BIGTOP/How+to+Contribute
+    2. Make sure your PR title starts with JIRA issue id, e.g., 'BIGTOP-3638: Your PR title ...'.
+-->
+
+### Description of PR
+
+
+### How was this patch tested?
+
+
+### For code changes:
+
+- [ ] Does the title or this PR starts with the corresponding JIRA issue id (e.g. 'BIGTOP-3638. Your PR title ...')?
+- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
```

Comment:
@yoda-mon I think you meant `s/ASF 2.0/ASF products/`. Dependencies are not necessarily licensed under Apache License 2.0.

Reply:
Thank you for the advice, I just remained the sentence from the source (hadoop template) .
How about replacing the sentence to `make sure that newly added files do not have any licensing issues. When in doubt refer to https://www.apache.org/licenses/`, from the [Bigtop contribution guide](https://cwiki.apache.org/confluence/display/BIGTOP/How+to+Contribute#HowtoContribute-Howtogeneratepatchesbyformattingpatchfiles).
More precisely I should add `Run mvn apache-rat:check to make sure that...` but it might sound a little bit strict.

Or simply replace to `Check [Bigtop contribution guide](https://cwiki.apache.org/confluence/display/BIGTOP/How+to+Contribute)`.

Reply:
> How about replacing the sentence to make sure that newly added files do not have any licensing issues. When in doubt refer to https://www.apache.org/licenses/

I'm +1 on this.

Reply:
Thank you for you, I changed the sentence to it.